### PR TITLE
fix generate is failed in windows

### PR DESCRIPTION
**Pull request by freezeChen**
#70

### CODE REVIEW

File: entc/cmd/entc/entc.go [hunk 1]
```diff
@@ -133,6 +133,7 @@ func loadGraph(path string, cfg gen.Config) (*gen.Graph, error) {
 	}
 	cfg.Schema = spec.PkgPath
 	cfg.Package = filepath.Dir(spec.PkgPath)
+	cfg.Package = strings.Replace(cfg.Package, "\\", "/", -1)
```
Comment (a8m) [inline entc/cmd/entc/entc.go]: Thanks.
Can you please add a comment here? Thanks

**Comment by alexsn** [comment]
Please use `filepath.ToSlash` instead of `strings.Replace`, in addition collapse both assignments into a single line (e.g. `cfg.Package = filepath.ToSlash(filepath.Dir(spec.PkgPath))`)

File: entc/cmd/entc/entc.go (hunk 1 above)
Comment (a8m) [reply entc/cmd/entc/entc.go]: Wait, I think the problem is with filepath -> path.

File: entc/cmd/entc/entc.go [hunk 2]
```diff
@@ -133,6 +133,7 @@ func loadGraph(path string, cfg gen.Config) (*gen.Graph, error) {
 	}
 	cfg.Schema = spec.PkgPath
 	cfg.Package = filepath.Dir(spec.PkgPath)
```
Comment (a8m) [inline entc/cmd/entc/entc.go]: From the `filepath godoc:
filepath.Dir:
> Dir calls Clean on the path and trailing slashes are removed.
filepath.Clean
> Finally, any occurrences of slash are replaced by Separator.

And `Separator` is set to `os.PathSeparator`. Can you try replace `filepath` with `path`? 
```suggestion
	cfg.Package = path.Dir(spec.PkgPath)
```

Comment (a8m) [reply entc/cmd/entc/entc.go]: OK, checked it now on Windows machine.

```go
println(path.Dir("a/b/c"))     // prints "a/b"
println(filepath.Dir("a/b/c")) // prints "a\b"
```

**Comment by a8m** [comment]
Closed with 7597f079129814c3ac37f823fae9d1b6f862e40b

Thanks for your contribution.

**Pull request closed without merging**